src/Components/Atoms/FAChip/FAChip.interface.tsx

```diff
@@ -0,0 +1,8 @@
+import { ChipProps } from "@mui/material";
+
+export interface IFAChip extends ChipProps {
+  label?: string;
+  testID: `chip-${string}`;
+  avatar?: any;
```

Comment:
change `any` to a specific type

Reply:
it looks like we have our first disagreement. an avatar could be a string, a number, an icon, a boolean.
kindly check the storybook and figma.